python/cloudtik/runtime/metastore/scripts/install.sh

```diff
@@ -76,6 +76,7 @@ function install_hive_metastore() {
           tar -zxf hive-standalone-metastore-${HIVE_VERSION}-bin.tar.gz && \
           mv apache-hive-metastore-${HIVE_VERSION}-bin hive-metastore && \
           rm hive-standalone-metastore-${HIVE_VERSION}-bin.tar.gz)
+      wget --no-check-certificate https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.38/mysql-connector-java-5.1.38.jar -P $METASTORE_HOME/lib/
```

Comment:
Please check other wget to as to the parameters for downloading.
We need to the download thing be clean designed at the console output.